src/components/twilio-voice-basic-call-control/public/twilio-voice-basic-call-control.js

```diff
@@ -1 +1,107 @@
-// TODO: Implement Web Component
+class TwilioVoiceBasicCallControl extends HTMLElement {
+  #call;
+  #callSid;
+  #conferenceSid;
```

Comment:
It seems like there are two different "unset" values for these. Later we set them to `null`, but here they are `undefined`. We should settle on `undefined` IMO.

Reply:
Makes sense. Any reason for `undefined` over `null`?

Reply:
Purely because leaving it "blank" sets the value to `undefined`. Consider the following:
```
const foo = null;
const bar = undefined;

typeof foo === 'object'; // confusing
typeof bar === 'undefined'; // less confusing

if (foo === null) { ... } // works
if (bar === undefined) { ... } // does not work, very confusing
```
So both of them have pretty confusing cons, and the benefit is that we just don't have to set things to `null` by hand things will be implicit `undefined`.